server/src/test/java/org/elasticsearch/indices/IndexingMemoryControllerTests.java

```diff
@@ -363,7 +363,7 @@ public void testTranslogRecoveryWorksWithIMC() throws IOException {
         for (int i = 0; i < 100; i++) {
             indexDoc(shard, Integer.toString(i), "{\"foo\" : \"bar\"}", XContentType.JSON, null);
         }
-        shard.close("simon says", false);
```

Comment:
Bah I wish we could have preserved this great reason! :)

Reply:
it's still there in `org.elasticsearch.indices.IndicesLifecycleListenerSingleNodeTests` :)